src/pages/admin.tsx

```diff
@@ -36,37 +52,69 @@ export default function AdminPage() {
   }, [user]);
 
   if (authorize === false) return <Custom404 />;
-  if (loading) return <Loading />;
 
   return (
     <CommonPage page={"Admin"} backBar={false}>
       <div className={"flex justify-center"}>
         <div className={`${isDesktop ? "w-2/3" : "w-full"}`}>
-          <div className="flex flex-col mb-8">
-            <h1 className="text-xl">{t("admin.skillsPending")}</h1>
-            {skills.Skill && (
-              <p className="opacity-50">
-                {
-                  skills.Skill.filter((field) => field.verified === false)
-                    .length
-                }{" "}
-                {t("search.result")}
-              </p>
-            )}
-          </div>
-          {skills.Skill.length > 0 &&
-            skills.Skill.filter((field) => field.verified === false).map(
-              (skill, index) => (
-                <NotificationPanel
-                  key={index}
-                  skill={{
-                    name: skill.name,
-                    skillId: skill.id,
-                    verified: skill.verified,
-                  }}
-                ></NotificationPanel>
-              )
-            )}
+          <SearchBar
+            initialValue={search}
+            value={search}
+            setSearch={setSearch}
+            placeholder={t("search.placeholder")}
+          />
+          {loading ? (
+            <div />
+          ) : (
```

Comment:
```suggestion
          {!loading && (
```

I suggest this to avoid unused div